plugins/com.google.cloud.tools.eclipse.appengine.newproject.test/src/com/google/cloud/tools/eclipse/appengine/newproject/StandardProjectWizardTest.java

```diff
@@ -16,16 +16,22 @@ public void setUp() {
     } catch (NullPointerException ex) {
       wizard = new StandardProjectWizard();
     }
+    wizard.init(null, null);
```

Comment:
You're not triggering the normal lifecycle of the wizard.  When a WizardDialog is opened on the Wizard, it calls Wizard.addPages() before rendering the controls on the active page.

You could do something like this:
```
WizardDialog wizardDialog = new WizardDialog(null, wizard);
wizardDialog.setBlockOnOpen(false);
wizardDialog.open();
Assert.assertNotNull("Wizard dialog should be open", wizardDialog.getShell());
// and then other checks